src/core/cgn.tf

```
@@ -122,3 +122,57 @@ module "cgn_cosmos_db" {
   account_name        = module.cosmos_cgn.name
 }
 
+
+## Blob storage due to legal backup
+#tfsec:ignore:azure-storage-default-action-deny
+// TODO: verify to use underscore or middlescore
+module "cgn-legalbackup-storage" {
+  source = "git::https://github.com/pagopa/azurerm.git//storage_account?ref=v1.0.79"
+
+  name                       = replace(format("%s_legalbackup_storage", local.project), "-", "")
+  account_kind               = "StorageV2"
+  account_tier               = "Standard"
+  account_replication_type   = var.legalbackup_account_replication_type
+  access_tier                = "Hot"
+  versioning_name            = "versioning"
+  enable_versioning          = var.legalbackup_enable_versioning
```

```suggestion
  enable_versioning          = var.cgn_legalbackup_enable_versioning
```
since variables are global, it's good practice add cgn suffix.